app/src/main/java/de/langerhans/odintools/service/BatteryLevelMonitorService.kt

```diff
@@ -0,0 +1,97 @@
+package de.langerhans.odintools.service
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.os.IBinder
+import androidx.core.app.NotificationCompat
+import dagger.hilt.android.AndroidEntryPoint
+import de.langerhans.odintools.R
+import de.langerhans.odintools.tools.BatteryLevelReceiver
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class BatteryLevelMonitorService @Inject constructor() : Service() {
```

Comment:
This might be a dumb question but can our accessibility service do the same work? Would spare us from having to run a foreground service.

Reply:
Great question! I haven't tested this yet. Will give it a try and report back.

Reply:
Resolved in 8d99eea